subjects/java/checkpoints/digitlen/README.md

```diff
@@ -0,0 +1,36 @@
+## Digitlen
+
+### Instructions
+
+In a file named `Digitlen.java` write a function `digitlen` that returns the length of the number passed as a parameter.
```

Comment:
How should the `-` be counted for negative numbers?

Reply:
yeah you are right is ambiguous, so i change it to return `the count of digits in the number...`